rkt.spec

```diff
@@ -7,14 +7,11 @@
 %global repo rkt
 
 %global import_path %{provider}.%{provider_tld}/%{project}/%{repo}
-#%global commit `git log | head -1 | cut -d' ' -f2
-%global commit 45cade58135d915ca1525f268369f9bef4d4fed6
+#%global commit % (git log | head -1 | cut -d' ' -f2)
```

Comment:
%%global is a better solution as you know it is/was a macro and all you need to do is to remove one % instead of adding one when you uncommented it out.